images/linux/scripts/installers/1804/powershellcore.sh

```diff
@@ -8,6 +8,8 @@
 source $HELPER_SCRIPTS/document.sh
 
 LSB_RELEASE=$(lsb_release -rs)
+
+sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-bionic-prod bionic main" > /etc/apt/sources.list.d/dotnetdev.list'
```

Comment:
The same repo is  installed in the repos.sh  script. Why do you put it again?

Reply:
string  has been removed.
commit: andy-mishechkin@aca1c5a